apps/test/unit/javalab/JavalabEditorTest.js

```diff
@@ -25,6 +25,7 @@ describe('Java Lab Editor Test', () => {
     defaultProps = {
       onCommitCode: () => {}
     };
+    window.appOptions = {level: {}};
```

Comment:
The window object gets shared between tests. To prevent odd errors when all tests get run together, you'll want to follow a pattern similar to this: https://github.com/code-dot-org/code-dot-org/blob/f1a94eab0ffe534ff8592e5f21a9c007edd30ba3/apps/test/unit/templates/instructions/InlineAudioTest.js#L30-L39